test/test_ops.py

```diff
@@ -397,6 +397,15 @@ def test_variant_consistency_jit(self, device, dtype, op):
                 if variant is None:
                     continue
 
+                # scripting and check_alias_analysis do not work with lambdas
+                # lambdas are typically used as a way to simulate methods without
+                # functional variants, so rely on the other variant for testing
+                # for now
+                if isinstance(variant, LambdaType):
```

Comment:
Should we just use this approach instead of adding metadata to the OpInfo?

Reply:
has_fake_function != is_lambda